integration/benchmark/cmd/ci/main.go

```diff
@@ -52,9 +54,14 @@ const (
 	metricport                  = uint(8080)
 	accessNodeAddress           = "127.0.0.1:3569"
 	pushgateway                 = "127.0.0.1:9091"
-	accountMultiplier           = 100
+	accountMultiplier           = 50
 	feedbackEnabled             = true
 	serviceAccountPrivateKeyHex = unittest.ServiceAccountPrivateKeyHex
+
+	// Auto TPS scaling constants
+	additiveIncrease       = 50
+	multiplicativeDecrease = 0.7
```

Comment:
you may want to use an even larger number, say 0.8 or 0.9, if you to narrow the bound

Reply:
fair enough-- I was using a pretty conservative constant from early TCP implementations, let's bump it up a bit.